shap/plots/_violin.py

```diff
@@ -37,48 +37,64 @@ def violin(
     color_bar=True,
     plot_size="auto",
     layered_violin_max_num_bins=20,
-    class_names=None,
-    class_inds=None,
     color_bar_label=labels["FEATURE_VALUE"],
     cmap=colors.red_blue,
+    color_bar_label_size=11,
```

Comment:
```suggestion
    color_bar_label_size=12,
```
to recover previous behaviour

Reply:
Fixed in [039a61e](https://github.com/shap/shap/pull/4119/commits/039a61e09f42745dfa8f5e2cef9fd4b10fd5c3e9).